surefire-integration-tests/src/test/resources/junit5/src/test/java/junit5/JUnit5Test.java

```diff
@@ -53,13 +53,13 @@ public void tearDown()
     @Test
     public void testSetUp()
     {
-        assertTrue( setUpCalled, "setUp was not called" );
+        Assertions.assertTrue( setUpCalled, "setUp was not called" );
```

Comment:
It's up to your taste:)
I like using it because especially in the assertion statements are verbose and if you read them in whole line then it sounds like sentence you read in news, very narrative. And therefore the class name does not have meaning to make it more verbose, it's just only a place where the function is stored.
The line width is limited to 120 chars which forces me to decide on how to type complete statement without breaking the line and without wrapping with nested statement.